test/com/google/javascript/jscomp/CodePrinterTest.java

```diff
@@ -139,6 +139,15 @@ public void testObjectLiteralWithSpread() {
     assertPrintSame("({...{...{}}})");
   }
 
+  @Test
+  public void testObjectLiteralWithComma() {
```

Comment:
Well done remembering to add test cases.
Thanks.

Reply:
Ah ah thank you. That's actually the bare minimum for me. If there are other kinds of tests that would be relevant, I'm happy to add more if you can point me to the right place.